src/js/components/reviewData/ReviewDataContent.jsx

```diff
@@ -192,10 +194,17 @@ export default class ReviewDataContent extends React.Component {
                                         Monthly submissions cannot be certified
                                     </div>
         }
-        if (this.checkAffiliations() || this.props.session.admin) {
+
+        if ((this.checkAffiliations() || this.props.session.admin ) && buttonClass != " btn-disabled") {
```

Comment:
Should all of these "if" statements be "else if"? I'm not sure, go through and check because you may be able to simplify them if they're else if